src/foam/nanos/dig/DUGRuleAction.js

```diff
@@ -44,19 +44,28 @@ foam.CLASS({
         agency.submit(x, new ContextAgent() {
           @Override
           public void execute(X x) {
-            HTTPSink sink = new HTTPSink(
-              getUrl(), 
-              getBearerToken(), 
-              getFormat(),
-              new foam.lib.AndPropertyPredicate(x, 
-                new foam.lib.PropertyPredicate[] {
-                  new foam.lib.ExternalPropertyPredicate(),
-                  new foam.lib.NetworkPropertyPredicate(), 
-                  new foam.lib.PermissionedPropertyPredicate()}),
-              true);
-              
+            var pm = new PM(DUGRuleAction.getOwnClassInfo().getId(), "dugrule." + dugRule.getId() + "." + dugRule.getName());
```

Comment:
No need to do this here, 
```suggestion
            var pm = new PM(DUGRuleAction.getOwnClassInfo().getId(), "dugrule.", dugRule.getId(), ".", dugRule.getName());
```
And PMs already combine names with ':' like a path.